Updated Forge images and resize documentation; added Windows and other 2.0 references (#593)
timescale/docs.timescale.com-content: getting-started/exploring-forge.md

<img class="main-content__illustration" src="https://assets.iobeam.com/images/docs/forge_images/timescale-forge-service-creation.png" alt="Set up a Timescale Forge service"/>

1. First, supply your service name (e.g., `acmecorp-test` or `acmecorp-dev`).
1. Next, choose your CPU and memory configuration, from (0.5 CPU, 2GB RAM) to (8 CPU, 32 GB RAM).
1. Select your storage requirements, from 25 GB to 1 TB.  Note with TimescaleDB compression, this is typically equivalent to 400 GB to over 16 TB of uncompressed storage (although compression rates can vary based on your data).
1. Note the estimated cost of running your chosen configuration. Feel free to [contact us][contact-timescale] if you would like to discuss pricing and configuration options best suited for your use case.
1. Click 'Create service' once your configuration is complete.

>:TIP:Don't worry if too much about the size settings that you choose initially. With Timescale Forge,
it's easy to modify both the compute (CPU/Memory) and storage associated with the service

> that you just created. As you get to know Timescale and how your data processing needs vary,

Reviewer: ```suggestion
that you just created. As you get to know TimescaleDB and how your data processing needs vary,
```

to add your credit card information. This will ensure a smooth transition after 
your trial period concludes.

## Resizing Compute and Storage in Timescale Forge [](forge-resize)

Timescale Forge allows you to resize compute (CPU/RAM) and storage independently at any time. This is extremely
useful when users have a need to increase storage (for instance) but not compute. The Timescale Forge 
console makes this very easy to do for any service.

Before you modify the compute or storage settings for a Forge Service, please note the following limitations
and when a change to these settings will result in momentary downtime.

**Storage**: Storage changes are applied with no downtime, typically available within a few seconds. Other
things to note about storage changes:
 * At the current time, storage can only be increased in size.
 * Storage size changes can only be made once every six (6) hours.
 * Storage can be modified in 25GB increments between 25GB-500GB, and then in 100GB increments between 500GB-1TB.

> **Compute**: Modifications to the compute settings of your service (up or down) can be applied at any time, however,

Reviewer: ```suggestion
**Compute**: Modifications to the compute size of your service (increases or decreases) can be applied at any time, however,
```

please note the following:
 * **_There will be momentary downtime_** while the compute settings are applied. In most cases, this downtime

> will be less than a minute.

Reviewer: ```suggestion
will be less than 30 seconds.
```
